src/Microsoft.TemplateEngine.Edge/Mount/Archive/ZipFileMountPointFactory.cs

```diff
@@ -58,6 +75,11 @@ public bool TryMount(IEngineEnvironmentSettings environmentSettings, IMountPoint
                     mountPoint = null;
                     return false;
                 }
+                if (skipMounting)
```

Comment:
I would suggest we try to open and then dispose. The file may exists but not be a valid archive.

Reply:
i.e. CanMount does not guarantee TryMount works.